src/index.ts

```diff
@@ -22,29 +22,74 @@ interface TelescopeNode {
   telescopicOut: TelescopicOutput;
 }
 
+/**
+ * Modes that designate what form the input text is in and should be interpreted as.
+ */
+enum TextMode {
+  Text = "text",
+  Html = "html",
+}
+const TextModeValues = Object.values(TextMode);
+
 interface Config {
   /**
-  * Character used to separate entries on the same level. Defaults to a single space (" ")
-  */
+   * Character used to separate entries on the same level. Defaults to a single space (" ")
+   */
   separator?: string;
   /**
-  * If true, allows sections to expand automatically on mouse over rather than requiring a click.
-  */
+   * If true, allows sections to expand automatically on mouse over rather than requiring a click. Defaults to false.
+   */
   shouldExpandOnMouseOver?: boolean;
+  /**
+   * A mode that designates what form the input text is in and should be interpreted as. Defaults to 'text'.
+   */
+  textMode?: TextMode;
 }
 
-const DefaultConfig: Config = {
-  separator: " ",   // character to divide items on the same indentation level
-  shouldExpandOnMouseOver: false, // whether text can be expanded on hover
-}
+type CreateTelescopicTextConfig = Pick<
+  Config,
+  "shouldExpandOnMouseOver" | "textMode"
+>;
 
 // time; recorded to prevent recursive text expansion on single hover
 let _lastHoveredTime = Date.now();
 
 // Internal recursive function to hydrate a node with a line object.
-function _hydrate(line: Content, node: any, shouldExpandOnMouseOver: boolean = false) {
+function _hydrate(
+  line: Content,
+  node: any,
+  config: CreateTelescopicTextConfig = {}
+) {
+  const { shouldExpandOnMouseOver, textMode = TextMode.Text } = config;
   let lineText = line.text;
 
+  function createLineNode(lineText: string) {
+    switch (textMode) {
+      case TextMode.Text:
+        // TODO: move this special character replacement functionality into a general purpose config
+        // passed in at the top-level.
+        if (lineText.includes("@Q ")) {
+          // if this is a quotation, turn it into a <blockquote> element
+          const [before, quote] = lineText.split("@Q ", 2);
+          const pre = document.createTextNode(before);
+          node.appendChild(pre);
+          const el = document.createElement("blockquote");
+          el.innerText = quote;
+          return el;
+        } else {
+          return document.createTextNode(lineText);
+        }
+
+      case TextMode.Html:
+        const newNode = document.createElement("span");
```

Comment:
by default wrapped this in a span 🤷  I guess could be an additional customization to specify what the default wrapping element is too

Reply:
I think it's a bit weird to _enforce_ a wrapping element in the input which is why I added this in the first place (i.e. i want to pass in `- hello <b>with bold</b>` rather than having to do `- <span> I'm wrapping my text in a span </span>`

Reply:
wish we had fragments in regular HTML 😔 looks like `span` can only have these as children: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content

this seems comprehensive enough for now? thoughts on adding an HTML class to this span so people can style it?

Reply:
yeah it's not ideal :( I think ideally we'd try to interpret it as HTML and then fall back to a default element (that defaults to span but could be configured) if not present (so then you could put whatever you wanted as the wrapper) but i wasn't sure how to do that optimistic determination work well. 

re: class hmm i think we should avoid adding new classes as much as possible and have them style via targeted styling i.e. `#telescope span...`

Reply:
i can update with being able to specify a default element tag